website/docs/components/form/select/partials/specifications/states.md

```diff
@@ -0,0 +1,49 @@
+## State
+
+#### Default
+
+Default
+
+![Select default with default state](/assets/components/select/select-state-default-default.png)
+
+Hover
+
+![Select default with hover state](/assets/components/select/select-state-default-hover.png)
+
+Focus
+
+![Select default with focus state](/assets/components/select/select-state-default-focus.png)
+
+#### Disabled
+
+Default
+
+![Select disabled with default state](/assets/components/select/select-state-disabled-default.png)
+
+Hover
+
+![Select disabled with hover state](/assets/components/select/select-state-disabled-hover.png)
+
+Focus
+
+![Select disabled with focus state](/assets/components/select/select-state-disabled-focus.png)
+
+#### Invalid
+
+Default
+
+![Select invalid with default state](/assets/components/select/select-state-invalid-default.png)
+
+Hover
+
+![Select invalid with hover state](/assets/components/select/select-state-invalid-hover.png)
+
+Focus
+
+![Select invalid with focus state](/assets/components/select/select-state-invalid-focus.png)
+
+### Disabled
+
+- Disabled fields are not editable by the user.
+- If the user doesn't need to review the data, consider using a hidden field instead.
```

Comment:
I would consider moving this section up to the previous disabled state section. 

I would also suggest adding the difference between Disabled and Readonly.

I also don't recommend hiding data that the user doens't need to review. How do we know it? If it is a form that the user is interacting with, they should at least see the disabled and readonly fields, for transparency.

Reply:
- I've reorganized the disabled content
- I don't see a readonly variant for the select, only for the other textinput fields unless I'm missing something? Readonly is listed as a potential attribute that can be passed in `...attributes` though
- I agree with you on this, but I think this goes deeper than just removing it here. Let's log it away and revisit holistically for all of the form elements.